src/list.hpp

```diff
@@ -106,6 +106,8 @@ class List {
         size_t valid_count;
     };
 
+    LinkViewRef get_linkview() const { return m_link_view; }
```

Comment:
The goal of avoiding having get_linkview() is just to ensure that people add any required functionality to List rather than being lazy and doing things on the linkview directly.